flowzone.yml

```diff
@@ -1956,7 +1956,7 @@ jobs:
         name: Check engine
         id: check_engine_18
         run: |
-          node_version="$(node --version)"
+          node_version="$(node --version | cut -d'.' -f1).x"
```

Comment:
I would rather we use the input that was provided to the Setup node step, instead of always assuming we want to truncate with .x

Reply:
I thought about that, so we would pass the input to `checkEngine` as well